### DISCUSSION TITLE
next-auth 4 blocks fetch request on RSC | Added Code and Live preview

### DISCUSSION THREAD

**Discussion by VladyslavL** [nextauthjs/next-auth#9224] [Help]
Next-auth 4 blocks a fetch request on the React Server Component but allows it on the React Client Component while a user is authenticated.

**Code here:**
https://github.com/VladyslavL/next-auth-rsc-fetch-bug

**How to reproduce**
LIVE: https://next-auth-rsc-fetch-bug.vercel.app/1

Login: test
Password: test

On the top of the page, you can see the fetch response on RSC and the response on RCC below.

Any idea why this is happening?

**Comment by VladyslavL** [comment]
updated with code and live preview

**Comment by wc4now** [comment]
December 9, 2023

I am having a problem of Live Server Preview Version 0.1.4 not opening in VS CODE Version 1.85.0
The computer is a Mac Mini M2 Pro Version Somoma 14.1.2
Please help ....I am doing a course and can't keep up.
